CONTRIBUTING.md

```diff
@@ -160,27 +168,43 @@ Uninstalling keras-nightly-2.5.0.dev2021032500:
 
 ## Run tests
 
+We use [Bazel](https://bazel.build/) to build and run the tests.
+
+### Run a test file
+
+For example, to run the tests in `keras/engine/base_layer_test.py`,
+we can run the following command at the root directory of the repo.
+
+```shell
+bazel test keras/engine:base_layer_test
+```
+
+`keras/engine` is the relative path to the directory containing the test file.
+`base_layer_test` is the test file name without the `.py` extension.
```

Comment:
This is the bazel test target name which can be different from the test file name. Please refer to the BUILD file for the corresponding test name.

Reply:
I have updated the description for the user to refer to the bazel BUILD file.